src/net/sourceforge/kolmafia/request/AdventureRequest.java

```diff
@@ -1029,18 +1031,17 @@ public static String parseEncounter(final String responseText) {
       }
     }
 
-    int boldIndex = responseText.indexOf("<b>", index);
-    if (boldIndex == -1) {
+    if (index == -1) {
```

Comment:
If you wanted, you could probably use `responseText.indexOf("<b", index)` as an approximation of the previous logic.

Reply:
That matches `<body` (although wouldn't with `index + 1`) and `<br` (which I think shouldn't occur in the places we look but can't guarantee)